app/workers/fast_track/hypertension_special_issue_manager.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module FastTrack
+  class HypertensionSpecialIssueManager
+    attr_accessor :submission
+
+    def initialize(submission)
+      @submission = submission
+    end
+
+    def add_special_issue
+      data = JSON.parse(submission.form_json)
+      disabilities = data['form526']['form526']['disabilities']
+      added = add_rrd_to_disabilities(disabilities)
+      data['form526']['form526']['disabilities'] = added
+      submission.update(form_json: JSON.dump(data))
+    end
+
+    def add_rrd_to_disabilities(disabilities)
+      disabilities.each do |da|
+        add_rrd(da) if da['diagnosticCode'] == 7101 && da['disabilityActionType'].downcase == 'increase'
+      end
+      disabilities
+    end
+
+    def add_rrd(disability)
+      rrd_hash = { 'code' => 'RRD', 'name' => 'Rapid Ready for Decision' }
```

Comment:
Could we use the new has syntax here?
```suggestion
      rrd_hash = { code: 'RRD', name: 'Rapid Ready for Decision' }
```